client/reader/start/main.jsx

```diff
@@ -55,8 +57,16 @@ const Start = React.createClass( {
 		page.redirect( '/' );
 	},
 
+	renderLoadingPlaceholders() {
+		const count = 4;
```

Comment:
I'll be the voice of dissent, at least for this particular value. 😈  

This isn't something that would change between deployment environments and moving it to config introduces a new layer of indirection that can break. Having it near where it's used is clear, easily understood, and unlikely to break in unexpected ways.